data/ka'het missions.txt

```diff
@@ -0,0 +1,220 @@
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Graveyard Label"
+	landing
+	invisible
+	source
+		attributes ka'het
+	on offer
+		event "graveyard label"
+		fail
+
+event "graveyard label"
+	galaxy "label graveyard"
+		sprite label/graveyard
+
+mission "Disabled Ka'het"
+	boarding
+	invisible
+	source
+		government "Ka'het" " Ka'het "
+	destination "Earth"
+	on offer
+		log "Successfully boarded a Ka'het, it appears to be possible by killing the organism without blowing up its artificial exoskeleton."
+		conversation
+			`Your ship slowly approaches the <origin>, now floating lifeless near you. As you move closer, you start noticing many holes in the weakest points of the hull and damages reaching what appears to be a large, slug-like being in the middle of the ship. While most of the ship's machinery doesn't seem to have received heavy damages, your weapons must have hit the slug's vital organs, killing it.`
+			`	You have no troubles docking with what seems to be its "exoskeleton", trying to decide what to do next: while the Ka'het is too different from a ship for you to command it, it's safe to say that you can now loot its outfits.`
+				decline
+
+mission "First Contact: Ka'het: Remnant 1A"
+	landing
+	minor
+	invisible
+	source
+		government "Remnant"
+	to offer
+		has "Graveyard Label: offered"
+		"remnant met taely" == 0
+		not "Remnant: Technology Available: offered"
+	on offer
+		conversation
+			`Do you want to ask the Remnant about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You ask around more than a dozen people, but none of them seems to be aware of what you're talking about. Either the information is too confidential to be given to a stranger like you, or you didn't ask the right people.`
+				decline
+
+mission "First Contact: Ka'het: Remnant 1B"
+	landing
+	minor
+	invisible
+	source "Viminal"
+	to offer
+		has "Graveyard Label: offered"
+		"remnant met taely" >= 1
+	on offer
+		conversation
+			`Do you want to ask Taely about the section of space south of the Ember Wastes, the Graveyard, and its inhabitants?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	You find Taely inside one of the outern hangars of the shipyard, apparently teaching a small group of people how to operate the basic repairs over an Albatross. You sit on a bench waiting for her to finish her lesson, and in a few minutes you see her walking towards you. "Hello, <first>!" she says. "Sorry, but I have nothing for you to do today." "Actually," you reply, "I'm here because I need to make you a few questions."`
```

Comment:
```suggestion
			`	You find Taely inside one of the outer hangars of the shipyard, apparently teaching a small group of people how to make basic repairs on an Albatross. You sit on a bench waiting for her to finish her lesson, after which she walks over to you. "Hello, <first>!" she says. "Is there something I can help you with?" `
			`	"I'm here because I need to ask you a few questions," you reply.`
```